lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -340,7 +337,7 @@ private AuditEvent buildProfileMatchedAuditEvent(
                         ipAddress);
         return new AuditEvent(
                 AuditEventTypes.IPV_GPG45_PROFILE_MATCHED,
-                componentId,
+                configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
```

Comment:
Is there a cost to these repeated calls to SSM? I guess not because it's cached?

Reply:
But yeah, we have the three minute cache so it's not like there's going to be loads more.